packages/types/src/localization.ts

```diff
@@ -484,20 +484,29 @@ type _LocalizationResource = {
     emailAddressPage: {
       title: LocalizationValue;
       verifyTitle: LocalizationValue;
+      formHint: LocalizationValue;
       emailCode: {
+        /**
+         * @deprecated UserProfile now only uses `emailAddressPage.formHint`.
+         */
```

Comment:
If I'm not missing something, this is a breaking change. Users will get the latest ClerkJS version but not the latest localization file, if they are using a different localization than `en-US`.

Reply:
I'll push some changes to keep the previous form hints instead of removing them, therefore it won't break for previous clerk-js versions.

Reply:
For newer clerk-js versions, the new localizations will have to be added for the "generic" form hint, which I think is reasonable considering it won't display the label:

![CleanShot 2025-01-21 at 09 01 43](https://github.com/user-attachments/assets/3f9c21ae-9bb9-445d-9235-a8b6aa5aad7c)

And then it'll keep showing the localized ones per strategy:

![CleanShot 2025-01-21 at 09 03 55](https://github.com/user-attachments/assets/e6f60f55-14df-4e30-9edb-19b1c0eb432c)

Reply:
Updated on https://github.com/clerk/javascript/pull/4406/commits/299abde85794cde015942f531bee82fab82dcd1c